sdk/search/azure-search-documents/azure/search/documents/_service/_indexers_client.py

```diff
@@ -0,0 +1,245 @@
+                :start-after: [START create_indexer]
+                :end-before: [END create_indexer]
+                :language: python
+                :dedent: 4
+                :caption: Create an Indexer
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexers.create(indexer, **kwargs)
+        return result
+
+    @distributed_trace
+    def create_or_update_indexer(self, indexer, name=None, **kwargs):
+        # type: (Indexer, Optional[str], **Any) -> Dict[str, Any]
+        """Creates a new indexer or updates a indexer if it already exists.
+
+        :param name: The name of the indexer to create or update.
+        :type name: str
+        :param indexer: The definition of the indexer to create or update.
+        :type indexer: ~search.models.Indexer
+        :return: The created Indexer
+        :rtype: dict
+        """
+        # TODO: access_condition
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+
+        if not name:
+            name = indexer.name
+        result = self._client.indexers.create_or_update(name, indexer, **kwargs)
+        return result
+
+    @distributed_trace
+    def get_indexer(self, name, **kwargs):
+        # type: (str, **Any) -> Dict[str, Any]
+        """Retrieves a indexer definition.
+
+        :param name: The name of the indexer to retrieve.
+        :type name: str
+        :return: The Indexer that is fetched.
+        :rtype: dict
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_indexer_operations.py
+                :start-after: [START get_indexer]
+                :end-before: [END get_indexer]
+                :language: python
+                :dedent: 4
+                :caption: Retrieve an Indexer
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexers.get(name, **kwargs)
+        return result
+
+    @distributed_trace
+    def get_indexers(self, **kwargs):
+        # type: (**Any) -> Sequence[Indexer]
+        """Lists all indexers available for a search service.
+
+        :return: List of all the Indexers.
+        :rtype: `list[dict]`
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_indexer_operations.py
+                :start-after: [START list_indexer]
+                :end-before: [END list_indexer]
+                :language: python
+                :dedent: 4
+                :caption: List all the Indexers
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexers.list(**kwargs)
+        return result.indexers
+
+    @distributed_trace
+    def delete_indexer(self, indexer, **kwargs):
+        # type: (Union[str, Indexer], **Any) -> None
+        """Deletes an indexer. To use only_if_unchanged, the Indexer model
```

Comment:
```suggestion
        """Deletes an indexer. To use access conditions, the Indexer model
```

Reply:
Have we had an API review?

Reply:
Not yet for Indexers operations, tho if you are asking about `only_if_unchanged` vs access conditions, @johanste  weighed in on that in #11116

Reply:
changed the documentation to match conditions